apps/web/package.json

```diff
@@ -92,6 +93,7 @@
     "@types/react-dom": "^18.2.7",
     "@types/uuid": "^9.0.2",
     "autoprefixer": "^10.4.15",
+    "babel-plugin-macros": "^3.1.0",
```

Comment:
are we using this plugin?

Reply:
Yeah, I probably need to get this out. the macros babel came in play when I had errors from .next not being in sync with the current installs of the application.